keyboards/xcstore/cc67/rules.mk

```diff
@@ -0,0 +1,11 @@
+SRC += matrix.c
+
+# Custom matrix
+CUSTOM_MATRIX = lite
+
+# RGB Matrix enabled
+WS2812_DRIVER = pwm
+
+# Wear-levelling driver
+EEPROM_DRIVER = wear_leveling
+WEAR_LEVELING_DRIVER = legacy
```

Comment:
Pretty sure that this isn't needed, as the stm32f4 uses the wear leveling driver by default, and uses the embedded flash rather than legacy driver. 

```suggestion
```

Reply:
Thanks for pointing it out, I didn't knew that STM32F4 already have `embedded_flash` support for wear leveling.